include/wimlib/list.h

```diff
@@ -323,33 +349,51 @@ hlist_add_head(struct hlist_node *n, struct hlist_head *h)
 }
 
 #define hlist_entry(ptr, type, member) container_of(ptr,type,member)
-
+#ifdef _MSC_VER
```

Comment:
This is the only redef needed if using `/std:clatest` with VS2022 (since sadly MSVC does not support `({ bunch; of; instructions; })` constructs).